datafusion/sql/src/statement.rs

```diff
@@ -253,6 +259,28 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 table_name,
                 filter,
             } => self.show_columns_to_plan(extended, full, table_name, filter),
+
+            Statement::Insert {
+                table_name,
+                columns,
+                source,
+                ..
```

Comment:
I suggest explicitly listing out the other  fields here (like 'selection') and throwing an error if they are `Some(..)`

Using `..` means if a user tries to use them they will be silently ignored